ruby-kafka.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency 'digest-crc'
+  spec.add_dependency "digest-murmurhash"
```

Comment:
What are the dependencies of this gem? The dependency policy for ruby-kafka is pretty strict, so for opt-in behavior the default would be to give an error message when the library cannot be detected and ask the user to install it themselves (would love optional dependencies in gems, but 🤷)

If the gem is really trivial and doesn't include C code then we can add it as a hard dep, but only then.

Reply:
Ah I see how the various compression codecs are used. Would a similar setup to that be ok here?

The gem is fairly trivial but it's basically just C code.

Reply:
If it's C code then I think it should be opt-in, so that would be a good change :+1:

Reply:
Done, thanks!